packages/bitcore-payid/package.json

```diff
@@ -15,21 +15,35 @@
     "start": "npm run build && node ts_build/src/index.js",
     "build": "npm run clean && tsc",
     "clean": "rm -rf ts_build",
-    "test": "npm run test:all",
-    "test:all": "npm run build && mocha 'ts_build/test/**/*.spec.js' --colors",
-    "test:unit": "npm run build && mocha 'ts_build/test/unit/**/*.spec.js' --colors",
-    "test:integration": "npm run build && mocha 'ts_build/test/integration/**/*.spec.js' --colors"
+    "test:all": "npm run build && ts-node ./node_modules/jasmine/bin/jasmine 'test/**/*.spec.ts'",
+    "test:unit": "npm run build && ts-node ./node_modules/jasmine/bin/jasmine 'test/unit/**/*.spec.ts'",
+    "test:integration": "npm run build && ts-node ./node_modules/jasmine/bin/jasmine 'test/integration/**/*.spec.ts' --config=jasmine.json",
+    "test:karma": "npm run build && karma start"
   },
   "dependencies": {
     "@payid-org/utils": "^1.2.0",
-    "bitcore-lib": "^8.22.2"
+    "asn1.js": "^5.4.1",
+    "bitcore-lib": "^8.23.1",
+    "bn.js": "^4.11.8",
+    "elliptic": "^6.5.3",
+    "hash.js": "^1.1.7"
   },
   "devDependencies": {
-    "@types/mocha": "^8.0.3",
+    "@types/jasmine": "^3.6.1",
```

Comment:
do you really need jasmine? we use mocha in most of bitcore.

Reply:
Switched to mocha. I had some trouble getting mocha to work with karma, but those issues seem to be resolved now!